bin/test-helm-e2e

```diff
@@ -9,19 +9,17 @@ if [ -z ${DOCKER_IMAGE_TAG+x} ]; then
   export DOCKER_IMAGE_TAG
 fi
 
-if [ -z ${TEST_NAMESPACE+x} ]; then
```

Comment:
Is that necessary? I like to export a fixed namespace when testing, so I don't have to change my `kubectl` statements every time...

Reply:
Yup. Reverted the change.